test_classes/pages/my_account_page.py

```diff
@@ -1,16 +1,34 @@
 from selenium.webdriver.common.by import By
-
 from .base_page import BasePage
-
 from test_classes.helpers.locator import Locator
 
 
 class MyAccountPage(BasePage):
-    page_title_locator = Locator(by=By.XPATH, value="//h2[.='My account']")
+    page_title_locator = Locator(by=By.XPATH, value="//h2[.='My Account']")
+    payment_methods_locator = Locator(by=By.LINK_TEXT, value='Payment methods')
```

Comment:
For these locators, please also make sure to add the element type on the variable name. For example payment_methods_link_locator, order_link_locator, addressess_link_locator.

This makes the locator variable more descriptive.